test/rest_api/task_manager_utils.py

```diff
@@ -67,3 +67,10 @@ def check_child_parent_relationship(rest_api, parent, tree_depth, depth=0):
         assert child["parent_id"] == parent["id"], f"Parent id of task with id {child_id} is not set"
         if depth + 1 < tree_depth:
             check_child_parent_relationship(rest_api, child, tree_depth, depth + 1)
+
+def clear_module_tasks(rest_api, module_name):
```

Comment:
drain_module_tasks would be better
and similar to other drain functions defined in scylla